### PR TITLE
#288 updated motivated section of html

### PR DESCRIPTION
**Pull request by Jayashree-panda**

### CODE REVIEW

File: html/html.md [hunk 1]
```diff
@@ -9,8 +9,14 @@ About 5.5 hours
 - "Debugging" lesson
 
 ### Motivation
-HTML is one of the building blocks required for making a webpage. Every single webpage uses HTML. To be a web developer, you need to know how to write HTML!
+- Hyper Text Markup Language is the markup language for creating web pages and web applications.
+- Almost all the modern web applications uses HTML for it's front-end. And if you wish to be a web developer,
```
Comment (manaswinidas) [inline html/html.md]: Can you please change 'use' instead of 'uses' and 'their' instead of 'it's'?

File: html/html.md [hunk 2]
```diff
@@ -9,8 +9,14 @@ About 5.5 hours
...
 
 ### Motivation
-HTML is one of the building blocks required for making a webpage. Every single webpage uses HTML. To be a web developer, you need to know how to write HTML!
+- Hyper Text Markup Language is the markup language for creating web pages and web applications.
+- Almost all the modern web applications uses HTML for it's front-end. And if you wish to be a web developer, 
+  you need to learn HTML.
+- HTML is really simple and easy to learn.
 
+  **Which companies use html?** Almost every company uses HTML for it's front-end. You can visit, [codecademy](http://www.codeacademy.com) and [hackerrank](http://www.hackerrank.com).
+
```
Comment (manaswinidas) [inline html/html.md]: Remove the extra line.

**Comment by manaswinidas** [comment]
Also, please change the description to 'Fixes #issuenumber' as this is the keyword to automatically close the issue being referenced.

**Comment by manaswinidas** [comment]
Please squash all your commits in the end

File: html/html.md (hunk 2 above)
Comment (Jayashree-panda) [reply html/html.md]: yup done

File: html/html.md [hunk 3]
```diff
@@ -10,13 +10,12 @@ About 5.5 hours
 
 ### Motivation
 - Hyper Text Markup Language is the markup language for creating web pages and web applications.
-- Almost all the modern web applications uses HTML for it's front-end. And if you wish to be a web developer, 
+- Almost all the modern web applications use HTML for their front-end. And if you wish to be a web developer, 
   you need to learn HTML.
 - HTML is really simple and easy to learn.
 
   **Which companies use html?** Almost every company uses HTML for it's front-end. You can visit, [codecademy](http://www.codeacademy.com) and [hackerrank](http://www.hackerrank.com).
```
Comment (manaswinidas) [inline html/html.md]: Remove the comma after visit

Comment (Jayashree-panda) [reply html/html.md]: done

**Comment by Jayashree-panda** [comment]
@alodahl done

File: express-js/express.md [hunk 4]
```diff
@@ -8,7 +8,9 @@
 - Node.js
 
 ### Motivation
-Express.js is one part of the MERN web development stack (MongoDB, Express.js, React.js, Node.js), we cover at Techtonica. Express.js allows you to quickly build a website served by Node.js.
+- Express.js is one part of the MERN web development stack (MongoDB, Express.js, React.js, Node.js), we cover at Techtonica. Express.js allows you to quickly build a website served by Node.js.
+- Express enables robust routing and improve the performance. 
+- It is flexible as there are numerous modules available on **npm** which can be directly plugged into Express.
```
[the remark this answers is not in the document]
Comment (Jayashree-panda) [reply express-js/express.md]: done

**Comment by Jayashree-panda** [comment]
@alodahl Hey! I am sorry to say I have not made different branches for 2 different issues. Can you help me with fixing it. Can I make PR again making different branches.

**Comment by alodahl** [comment]
@Jayashree-panda , this is a great opportunity to learn how to work with branches!  The next time you take an issue, make a new local git branch before you start saving and committing your changes.  Then when you are done, push up that branch and make a PR.  
After that makes sense, you can practice working on multiple branches for multiple issues at the same time.  Git flow is pretty easy to look up, but also take advantage of the #git_github channel on slack ... lots of people are excited to help you learn!

**Pull request merged**